src/Symfony/Component/Form/Tests/Extension/Validator/Type/FormTypeValidatorExtensionProvider.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Symfony\Component\Form\Tests\Extension\Validator\Type;
+
+trait FormTypeValidatorExtensionProvider
```

Comment:
Why split this in a trait? should be merged with the test class to me.

Reply:
This is one of the code rules I'm used to. Test file contain test logic, provider-traits contain test cases. In some cases, a provider-trait can be huge, especially for complex functional tests with many scenarios.

If necessary for the Symfony code style, I will merge this with a test class.

Reply:
please merge with the class yes.

Reply:
done